File: Algorithms/01-Warmup/02-compare-the-triplets/02.py
#!/usr/bin/env python


def solve(alice, bob):
    total = [0, 0]
    for x in range(0, len(alice)):
        if alice[x] > bob[x]:
            total[0] += 1
        elif alice[x] < bob[x]:
            total[1] += 1
    return total


a = [int(x) for x in input().strip().split(' ')]
b = [int(x) for x in input().strip().split(' ')]
result = solve(a, b)
print("%d %d" % (result[0], result[1]))

File: Algorithms/01-Warmup/02-compare-the-triplets/02.rb
#!/usr/local/bin/ruby -w

def solve(alice, bob)
    total = [0, 0]
    (0..2).each do |x|
        if alice[x] > bob[x]
            total[0] += 1
        elsif alice[x] < bob[x]
            total[1] += 1
        end
    end
    total
end

a = gets.strip.split(" ").map(&:to_i)
b = gets.strip.split(" ").map(&:to_i)
result = solve(a, b)
puts "#{result[0]} #{result[1]}"
